app/cli/internal/policydevel/lint.go

```diff
@@ -192,10 +193,14 @@ func (p *PolicyToLint) validateYAMLFile(file *File) {
 		outYAML, err := protoyaml.Marshal(&policy)
 		if err != nil {
 			p.AddError(file.Path, fmt.Sprintf("failed to marshal updated YAML: %v", err), 0)
-		} else if err := os.WriteFile(file.Path, outYAML, 0600); err != nil {
-			p.AddError(file.Path, fmt.Sprintf("failed to save updated file: %v", err), 0)
 		} else {
-			file.Content = outYAML
+			// Force 2-space indentation, protoyaml.Marshal uses 4 spaces by default and lacks
+			outYAML = bytes.ReplaceAll(outYAML, []byte("    "), []byte("  "))
```

Comment:
Isn't there are better way to do this? I've seen protoyaml has an `Indent` option. This should work:
```
outYaml, err := protoyaml.MarshalOptions{Indent:2}.Marshal(&policy)
```